[BOO] Adding more build and triage details to boo driver readme (#732)
iree-org/iree-turbine: iree/turbine/kernel/boo/conv_exports/README.md

> - set `IREE_PY_RUNTIME=tracy` in your environment.

Reviewer: IIRC we should delete this line as well.

- You may need to configure cmake with `-DTRACY_DELAYED_INIT=ON -DTRACY_MANUAL_LIFETIME=ON` to work around issues with the tracy client being closed before traces are captured.

> - Make sure `PYTHONPATH` is export and point to your local built of iree compiler and runtime python bindings.

Reviewer: ```suggestion
- Make sure `PYTHONPATH` is set to your locally built compiler and runtime python bindings. E.g. `source <build-dir>/.env && export PYTHONPATH`.
```

Author: That is more clear! I don't actually know .env file exists in the iree build!

> - build `iree-tracy-capture`. Use `-DIREE_BUILD_TRACY=ON`, build IREE compiler and export it from `<build-dir>/tracy` to the PATH environment variable.

Reviewer: ```suggestion
- build `iree-tracy-capture`. Use `-DIREE_BUILD_TRACY=ON` when building IREE, and include `<build-dir>/tracy` in your `PATH` environment variable.
```
